src/Symfony/Component/Security/Core/Authorization/UserAuthorizationCheckerInterface.php

```diff
@@ -17,6 +17,8 @@
  * Interface is used to check user authorization without a session.
  *
  * @author Nate Wiebe <nate@northern.co>
+ *
+ * @deprecated since Symfony 7.4, implement the UserGuestAuthorizationCheckerInterface instead.
```

Comment:
do we have to deprecate? it might be better not to?

Reply:
Dunno, others might have an opinion :) (please mind fabbot, it still has spotted a few things)